gh-pages/content/overview/toolchain.md

```diff
@@ -5,34 +5,43 @@
-| -------------- | ------------ | --------------------------------------------------------------------- |
-| [jsii]         | Stable       | Compiles TypeScript to jsii module                                    |
-| [jsii-pacmak]  | Stable       | Creates ready-to-publish language-specific packages from jsii modules |
-| [jsii-reflect] | Stable       | Strong-typed reflection library for jsii type systems                 |
-| [jsii-diff]    | Stable       | API backwards compatibility checker                                   |
-| [jsii-rosetta] | Experimental | Transpile code snippets (in docs) from TypeScript to jsii languages   |
-| [jsii-config]  | Experimental | Interactive tool for generating jsii configuration                    |
-| [jsii-release] | Community    | Publishes jsii modules to all supported package managers              |
-| [jsii-srcmak]  | Community    | Generates relocatable source code in jsii languages from typescript   |
-| [jsii-docgen]  | Community    | Generates markdown API documentation for jsii modules                 |
+| Name            | Release | Stability      | Description                                                           |
+| --------------- | ------- | -------------- | --------------------------------------------------------------------- |
+| [jsii1]         | `1.x`   | [Maintenance]  | Compiles TypeScript to jsii module (TypeScript 3.9 Syntax)            |
+| [jsii]          | `5.0.x` | GA             | Compiles TypeScript to jsii module (TypeScript 5.0 Syntax)            |
+| [jsii-pacmak]   | `1.x`   | GA             | Creates ready-to-publish language-specific packages from jsii modules |
+| [jsii-reflect]  | `1.x`   | GA             | Strong-typed reflection library for jsii type systems                 |
+| [jsii-diff]     | `1.x`   | GA             | API backwards compatibility checker                                   |
+| [jsii-rosetta1] | `1.x`   | [Maintenance]  | Transpile code snippets (in docs) from TypeScript to jsii languages   |
+| [jsii-rosetta]  | `5.0.x` | Experimental   | Transpile code snippets (in docs) from TypeScript to jsii languages   |
+| [jsii-config]   | `1.x`   | Experimental   | Interactive tool for generating jsii configuration                    |
+| [jsii-release]  | `1.x`   | Community      | Publishes jsii modules to all supported package managers              |
+| [jsii-srcmak]   | `1.x`   | Community      | Generates relocatable source code in jsii languages from typescript   |
+| [jsii-docgen]   | `1.x`   | Community      | Generates markdown API documentation for jsii modules                 |
 
 ??? question "Stability Definitions"
-    - **Stable**: Projects that comply with the [Semantic Versioning][semver] specification, and will hence not change
-      behavior or receive other breaking changes across minor and patch version bumps.
+    - **GA**: Projects that are deemed *Generally Available* and for which customers can expect full support, including
```

Comment:
```suggestion

    - **GA**: Projects that are deemed *Generally Available* and for which customers can expect full support, including
```

The format looks strange when in preview (not a list). I am guessing it's a missing new line here.

Reply:
Yea... this is the specific markdown flavor that mkdoc uses... It's mostly compatible with GFM, but there's some minute differences... I did run a preview with `mkdoc serve` locally and it looked all dandy... This is a case where you can't fully trust the GitHub render preview.